float64 refSteeringRate
float64 refSteeringAngle
float64 steeringDutyCycleLimit
float64 refVelocity
float64 refAcceleration
bool autonomyEnabled